src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -170,12 +171,41 @@ export default function ResearcherInfo(props) {
             please do not list External Collaborators or Internal Collaborators at a PI or equivalent 
             level here.`
           ),
-          button({
-            type: 'button', // default button element type inside a form is "submit".
-            className: 'button button-white btn-xs',
-            style: { marginTop: 25 },
-            onClick: () => {}
-          }, ['Add Collaborator'])
+          // non functional button
+          // button({
+          //   id: 'addNewCollaborator',
+          //   type: 'button', // default button element type inside a form is "submit".
+          //   className: 'button button-white',
+          //   style: { marginTop: 25 },
+          //   onClick: () => {}
+          // }, ['Add Collaborator']),
+          // old style button that adds a new collaborator
+          h(CollaboratorList_new, {
+            formFieldChange,
+            collaborators: labCollaborators,
```

Comment:
I can't find where `labCollaborators` is being read from the response, and similarly for the other collaborators. Is that intentional? It would be difficult to test how this works with an existing collaborator without that population happening.

Reply:
Ignore the above. I can see that the page is loading all of the collaborators that exist after restarting/refreshing